scripts/ui/inventory/inventory.gd

```diff
@@ -0,0 +1,141 @@
+extends UserInterface
+
+const SlotClass = preload("res://scripts/ui/inventory/slot.gd")
+@onready var inventory_slots = $VBoxContainer2/PanelContainer2/HBoxContainer/GridContainer
+@onready var equip_slots = $VBoxContainer2/PanelContainer2/HBoxContainer/VBoxContainer/EquipSlots.get_children()
+@onready var tooltip = $Tooltip
+var t = null
+
+# Called when the node enters the scene tree for the first time.
+func _ready() -> void:
+	var slots = inventory_slots.get_children()
+	PlayerInventory.inventory_active_item_updated.connect(self.update_active_item_label)
+	for i in range(slots.size()):
+		slots[i].gui_input.connect(slot_gui_input.bind(slots[i]))
+		PlayerInventory.inventory_active_item_updated.connect(slots[i].refresh_style)
+		slots[i].inventory_slot_index = i
+		slots[i].slot_type = SlotClass.SlotType.INVENTORY
+	for i in range(equip_slots.size()):
+		equip_slots[i].gui_input.connect(slot_gui_input.bind(equip_slots[i]))
+		equip_slots[i].equip_slot_index = i
+	equip_slots[0].slot_type = SlotClass.SlotType.HELMET
```

Comment:
Are we creating the inventory slots at runtime? Otherwise we should be able to set these in export variables. All good for now.